main/webapp/modules/core/scripts/index/parser-interfaces/json-parser-ui.js

```diff
@@ -294,6 +294,8 @@ Refine.JsonParserUI.prototype._updatePreview = function() {
 
         new Refine.PreviewTable(projectData, self._dataContainer.unbind().empty());
       }, 100);
-    }
+	} else {
+		self._progressContainer.hide();
+	}
```

Comment:
@WaltonG: there is indenting issues here. Can it be that you use `\t` chars?

Reply:
Am looking into it